app/javascript/components/Shared/WhatHappensInThisResearchSessionAbridged.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Output from "./Output";
+import TakePartIn from "./TakePartIn";
+
+const WhatHappensInThisResearchSessionAbridged = props => {
```

Comment:
It seems this file is never used in the `/research-sessions/my-session/preview` state. Could you explain what this is used for?